script/incremental_build.sh

```diff
@@ -49,18 +49,18 @@ PLUGIN_SHARDING=($PLUGIN_SHARDING)
 
 if [[ "${BRANCH_NAME}" == "master" ]]; then
   echo "Running for all packages"
-  (cd "$REPO_DIR" && $PUB global run flutter_plugin_tools "${ACTIONS[@]}" --exclude="$ALL_EXCLUDED" ${PLUGIN_SHARDING[@]})
+  (cd "$REPO_DIR" && activate_plugin_tool && $PUB global run flutter_plugin_tools "${ACTIONS[@]}" --exclude="$ALL_EXCLUDED" ${PLUGIN_SHARDING[@]})
 else
   # Sets CHANGED_PACKAGES
   check_changed_packages
 
   if [[ "$CHANGED_PACKAGES" == "" ]]; then
     echo "No changes detected in packages."
     echo "Running for all packages"
-    (cd "$REPO_DIR" && $PUB global run flutter_plugin_tools "${ACTIONS[@]}" --exclude="$ALL_EXCLUDED" ${PLUGIN_SHARDING[@]})
+    (cd "$REPO_DIR" && activate_plugin_tool && $PUB global run flutter_plugin_tools "${ACTIONS[@]}" --exclude="$ALL_EXCLUDED" ${PLUGIN_SHARDING[@]})
   else
     echo running "${ACTIONS[@]}"
-    (cd "$REPO_DIR" && $PUB global run flutter_plugin_tools "${ACTIONS[@]}" --plugins="$CHANGED_PACKAGES" --exclude="$ALL_EXCLUDED" ${PLUGIN_SHARDING[@]})
+    (cd "$REPO_DIR" && activate_plugin_tool && $PUB global run flutter_plugin_tools "${ACTIONS[@]}" --plugins="$CHANGED_PACKAGES" --exclude="$ALL_EXCLUDED" ${PLUGIN_SHARDING[@]})
```

Comment:
Ideally we should abstract `pub global run` to a utility that does the activate then the run, instead of having to remember to put this on every call.

Reply:
I didn't go that far since we will be moving the plugins tool over to this repo.

Reply:
Done